languages.yaml

```diff
@@ -57,7 +57,9 @@ it:
     - auanasgheps
     - xraver
 ka:
-  nativeName: ಕನ್ನಡ
+  nativeName: ქართული
+  leaders:
+    - hertzg
```

Comment:
We want to see at least 2 contributions before accepting a language leader – [guidelines](https://voice.home-assistant.io/docs/contributing/language-leaders)
```suggestion
```